Tests/test_align_substitution_matrices.py

```diff
@@ -120,7 +145,7 @@ def test_nucleotide_freq(self):
         path = os.path.join("Align", "ecoli.txt")
         frequencies = substitution_matrices.read(path)
         self.assertEqual(frequencies.alphabet, nucleotide_alphabet)
-        self.assertEqual(frequencies.shape, (len(nucleotide_alphabet), ))
+        self.assertEqual(frequencies.shape, (len(nucleotide_alphabet),))
```

Comment:
I think that final comma can be removed (here and 3 more cases lower down - very obvious looking at the diff on GitHub).

Reply:
I believe I got all of them.

Reply:
As noted later, these were deliberate for expressing a single element tuple